src/components/pages/Offers/Offer/OfferDetails/__specs__/OfferCreationForPro.spec.jsx

```diff
@@ -386,7 +388,7 @@ describe('offerDetails - Creation - pro user', () => {
           userEvent.type(descriptionInput, 'Ma jolie description')
 
           // then
-          expect(await screen.queryAllByText('Ma jolie description')).toHaveLength(2)
+          expect(screen.queryAllByText('Ma jolie description')).toHaveLength(2)
```

Comment:
Ici on va appliquer la même idée, mais le `within` est obligatoire puisque le `getByText` ciblera aussi bien le `textarea` que le texte de l'aperçu.
```suggestion
          const offerPreview = screen.getByTestId('offer-preview-section')
          expect(within(offerPreview).getByText('Ma jolie description')).toBeInTheDocument()
```

Les `queryBy` sont à réserver pour les cas où on s'attend à ne pas trouver l'élément (souvent suivis d'un `.not.toBeInTheDocument()`)